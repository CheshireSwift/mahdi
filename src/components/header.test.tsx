import * as React from 'react';
import { render } from 'enzyme';

import Header from './header';

describe('the header', () => {
  it('matches the snapshot', () => {
    expect(render(<Header siteTitle="Title Goes Here" />)).toMatchSnapshot();
  });
});
