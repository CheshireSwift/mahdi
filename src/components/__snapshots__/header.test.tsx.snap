// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the header matches the snapshot 1`] = `
initialize {
  "0": Object {
    "attribs": Object {
      "style": "background:rebeccapurple;margin-bottom:1.45rem",
    },
    "children": Array [
      Object {
        "attribs": Object {
          "style": "margin:0 auto;max-width:960px;padding:1.45rem 1.0875rem",
        },
        "children": Array [
          Object {
            "attribs": Object {
              "style": "margin:0",
            },
            "children": Array [
              Object {
                "attribs": Object {
                  "aria-current": "page",
                  "class": "",
                  "href": "/",
                },
                "children": Array [
                  Object {
                    "data": "Title Goes Here",
                    "next": null,
                    "parent": [Circular],
                    "prev": null,
                    "type": "text",
                  },
                ],
                "name": "a",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": null,
                "parent": [Circular],
                "prev": null,
                "type": "tag",
                "x-attribsNamespace": Object {
                  "aria-current": undefined,
                  "class": undefined,
                  "href": undefined,
                },
                "x-attribsPrefix": Object {
                  "aria-current": undefined,
                  "class": undefined,
                  "href": undefined,
                },
              },
            ],
            "name": "h1",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": null,
            "parent": [Circular],
            "prev": null,
            "type": "tag",
            "x-attribsNamespace": Object {
              "style": undefined,
            },
            "x-attribsPrefix": Object {
              "style": undefined,
            },
          },
        ],
        "name": "div",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": null,
        "parent": [Circular],
        "prev": null,
        "type": "tag",
        "x-attribsNamespace": Object {
          "style": undefined,
        },
        "x-attribsPrefix": Object {
          "style": undefined,
        },
      },
    ],
    "name": "div",
    "namespace": "http://www.w3.org/1999/xhtml",
    "next": null,
    "parent": null,
    "prev": null,
    "root": Object {
      "attribs": Object {},
      "children": Array [
        [Circular],
      ],
      "name": "root",
      "namespace": "http://www.w3.org/1999/xhtml",
      "next": null,
      "parent": null,
      "prev": null,
      "type": "root",
      "x-attribsNamespace": Object {},
      "x-attribsPrefix": Object {},
    },
    "type": "tag",
    "x-attribsNamespace": Object {
      "style": undefined,
    },
    "x-attribsPrefix": Object {
      "style": undefined,
    },
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;
